core/amber/src/main/python/core/architecture/managers/channel_marker_manager.py

```diff
@@ -0,0 +1,65 @@
+from collections import defaultdict
+from typing import Set, Dict
+
+from proto.edu.uci.ics.amber.core import ActorVirtualIdentity, ChannelIdentity
+from proto.edu.uci.ics.amber.engine.architecture.rpc import (
+    ChannelMarkerPayload,
+    ChannelMarkerType,
+)
+
+
+class ChannelMarkerManager:
```

Comment:
do we still need to reserve marker name for other things? can we simply change channel marker to marker? maybe in a future PR?

Reply:
in a future PR, we will change this to Control Message.